x/cwgrant/types/keys.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+import "cosmossdk.io/collections"
+
+const ModuleName = "cwgrant"
```

Comment:
nit: i feel the module name is too abstract. maybe something like `cw-feegrant` or `cwfees`